lib/ansible/modules/cloud/vmware/vmware_guest_file_operation.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Stéphane Travassac <stravassac () gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: vmware_guest_file_operation
+short_description: Files operation in a VMware guest operating system without network
+description:
+    - Module allows user to copy and fetch file and create,delete directory in the guest operating system.
```

Comment:
I'd change this to `Module to copy a file to a VM, fetch a file from a VM and create or delete a directory in the guest OS.`